src/OpenTelemetry.AutoInstrumentation/Instrumentations/GraphQL/GraphQLCommon.cs

```diff
@@ -45,42 +40,23 @@ internal class GraphQLCommon
 
     private static readonly ILogger Log = OtelLogging.GetLogger();
 
-    internal static Activity CreateActivityFromValidate(IDocument document)
-    {
-        Activity activity = null;
-
-        try
-        {
-            var tags = new GraphQLTags();
-            activity = ActivitySource.StartActivityWithTags(ValidateOperationName, tags);
-
-            tags.Source = document.OriginalQuery;
-        }
-        catch (Exception ex)
-        {
-            Log.Error(ex, "Error creating or populating scope.");
-        }
-
-        return activity;
-    }
-
     internal static Activity CreateActivityFromExecuteAsync(IExecutionContext executionContext)
     {
         Activity activity = null;
 
         try
         {
-            string source = executionContext.Document.OriginalQuery;
+            string query = executionContext.Document.OriginalQuery;
             string operationName = executionContext.Operation.Name;
-            string operationType = executionContext.Operation.OperationType.ToString();
+            string operationType = executionContext.Operation.OperationType.ToString().ToLowerInvariant();
```

Comment:
Double-checking: no chance of executionContext properties being null here, correct?